src/libraries/System.Transactions.Local/tests/CloneTxTests.cs

```diff
@@ -165,6 +165,7 @@ public void Run(CloneType cloneType, IsolationLevel isoLevel, bool forcePromote,
         [InlineData(CloneType.BlockingDependent, IsolationLevel.Snapshot, false, TransactionStatus.Aborted)]
         [InlineData(CloneType.BlockingDependent, IsolationLevel.Chaos, false, TransactionStatus.Aborted)]
         [InlineData(CloneType.BlockingDependent, IsolationLevel.Unspecified, false, TransactionStatus.Aborted)]
+        [PlatformSpecific(~TestPlatforms.Browser)] // Cannot wait on monitors on this runtime.
```

Comment:
-> PlatformDetection.IsThreadingSupported

Reply:
Addressed in recent commits